environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/model/parameters/azure/AzureCredentialCertificateResponse.java

```diff
@@ -0,0 +1,71 @@
+package com.sequenceiq.environment.api.v1.credential.model.parameters.azure;
+
+import java.io.Serializable;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.common.api.credential.AppCertificateStatus;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("AzureCertificateResponse")
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public class AzureCredentialCertificateResponse implements Serializable {
+
+    @ApiModelProperty
+    private AppCertificateStatus status;
+
+    @ApiModelProperty
+    private Long expiration;
+
+    @ApiModelProperty
+    private String base64;
```

Comment:
This is the way it makes sense on the API, since they are the property of the certificate:

```
 "azureCredentialProperties": {
 "authenticationType": "CERTIFICATE",
 "certificate": {
     "status": "KEY_GENERATED",
     "expiration": 1660743294,
     "sha512": "4cf184fa9e45..ab4ca5d5bc0e306",
     "base64": "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUNvekNDQVl1Z0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFEREFwam………pXRnJNQjRYRFRJeU1EZ3lOakV6TkRNMU1Wb1hEVEl6TURneU5qRXpORE0xTVZvd0ZURVRNQkVHQTFVRQpBeE1LWTJ4dmRXUmljbVZoYXpDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBSmoxCnZqTGMyTGFnTnBPaVNUNGs2R0VBRn="
  }
```

But any suggestions are welcome.